modules/web/src/app/cluster/details/cluster/node-list/template.html

```diff
@@ -424,6 +424,21 @@
                 {{element.spec.cloud.azure.osDiskSize}} GB
               </div>
             </km-property>
+
+            <ng-container *ngIf="element.spec.cloud.baremetal?.tinkerbell">
+              <km-property *ngIf="element.spec.cloud.baremetal.tinkerbell.hardwareRef?.Name">
+                <div key>Hardware Name</div>
+                <div value>{{ element.spec.cloud.baremetal.tinkerbell.hardwareRef?.Name }}</div>
+              </km-property>
+              <km-property *ngIf="element.spec.cloud.baremetal.tinkerbell.hardwareRef?.Namespace">
+                <div key>Hardware Namespace</div>
+                <div value>{{ element.spec.cloud.baremetal.tinkerbell.hardwareRef?.Namespace }}</div>
+              </km-property>
+              <km-property>
```

Comment:
do we need a `ngIf` directive also here ?

Reply:
No, OS Image URL is a mandatory field so value will be there.